python/dgl/nn/pytorch/conv/gatedgcnconv.py

```diff
@@ -0,0 +1,171 @@
+"""Torch Module for GatedGCN layer"""
+# pylint: disable= no-member, arguments-differ, invalid-name, cell-var-from-loop
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+from .... import function as fn
+
+
+class GatedGCNConv(nn.Module):
+    r"""GatedGCN: Residual Gated Graph ConvNets
+    <https://arxiv.org/pdf/1711.07553v2.pdf>`
+
+    .. math::
+        e_{ij}^{l+1}=D^l h_{i}^{l}+E^l h_{j}^{l}+C^l e_{ij}^{l}
+
+        norm_{ij}=\Sigma_{j\in N_{i}} \sigma\left(e_{ij}^{l+1}\right)+\varepsilon
+
+        \hat{e}_{ij}^{l+1}=\sigma(e_{ij}^{l+1}) / norm_{ij}
+
+        h_{i}^{l+1}=A^l h_{i}^{l}+\Sigma_{j \in N_{i}} \hat{e}_{ij}^{l+1} \odot B^l h_{j}^{l}
+
+    where h_{i}^{l} are node features of layer l, \Sigma is sigmoid function,
+    \varepsilon is miniterm, A^l, B^l, C^l, D^l, E^l are linear layers.
```

Comment:
Where is \varepsilon defined in the original paper?

Reply:
add missing paper: Benchmarking Graph Neural Networks